Packs/Forcepoint/Scripts/FPDeleteRule/FPDeleteRule_test.py

```diff
@@ -0,0 +1,38 @@
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
+
```

Comment:
Missing test(s) for mutual exclusion of `tritonsystem` and `remoteaccessname`

Reply:
Why do we need this? The coverage is 70% it does not need to be 100%. 
This test will not find any new issue. the test will use 2 mockers to test 2 lines that were mocked. It will not check any logic!

Reply:
@dantavori

Reply:
Discussed with Dan, and He told me that a unit test for this is not needed.